### DISCUSSION TITLE
Server user should not have login access

### DISCUSSION THREAD

**Discussion by tmccaff** [GameServerManagers/LinuxGSM#3829] [Ideas]
Not sure if mentioned but a better way is not to add a user for login ssh access better security

Example:

instead of adduser sdtdserver then ask for password

should be done by this:

useradd sdtdserver -m -s /usr/sbin/nologin (Debian)

to now login into that user

su -s /bin/bash sdtdserver

**Comment by Claiyc** [comment]
imho you should restrict ssh access to specific users only in general (via AllowedUsers in sshd_config)

**Reply by tmccaff** [reply]
I am saying with this approach it's more secure.

**Reply by tmccaff** [reply]
with the username default as is will able to login, etc. With the approach above you can't login to shell without using su -s
